otherlibs/unix/unix.mli

```diff
@@ -1346,13 +1370,13 @@ val domain_of_sockaddr: sockaddr -> socket_domain
 (** Return the socket domain adequate for the given socket address. *)
 
 val socketpair :
-     ?cloexec:bool -> socket_domain -> socket_type -> int ->
-                                                 file_descr * file_descr
-(** Create a pair of unnamed sockets, connected together.
-   See {!set_close_on_exec} for documentation on the [cloexec]
-   optional argument. *)
+  ?cloexec: (* thwart toosl/unlabel *) bool ->
+    socket_domain -> socket_type -> int ->
+    file_descr * file_descr
+(** Create a pair of unnamed sockets, connected together. *)
```

Comment:
The reference to `cloexec` has been lost

Reply:
Fixed.